react/features/base/dialog/components/native/BaseDialog.js

```diff
@@ -18,6 +19,11 @@ import { brandedDialog as styles } from './styles';
 export type Props = {
     ...AbstractProps,
 
+    /**
+     * The color-schemed stylesheet of the feature.
+     */
+    _colorSchemedStyles: StyleType,
```

Comment:
same as before, let's call this `_styles`

Reply:
Similar, and here it's even more visible: I don't want to get to a confusion where we have ```styles``` imported from ```styles.js``` and ```_styles``` resolved from Redux